docs/bestpractices/quizdesign.md

```diff
@@ -1,4 +1,4 @@
-![HPI Logo](img/HPI_Logo.png)
+![HPI Logo](../img/HPI_Logo.png)
 # Quiz Design
 ## General Remarks
```

Comment:
I cannot comment below, but would like to put it here:

 Documentation file 'bestpractices/quizdesign.md' contains a link to 'bestpractices/(https:/www.utexas.edu/academic/ctl/assessment/iar/students/plan/method/exams-mchoice-write.php)' which is not found in the documentation files.

The double paranthesis are too much. It should be `[good advice](https://www.utexas.edu/academic/ctl/assessment/iar/students/plan/method/exams-mchoice-write.php)`

Reply:
The link is fixed. Next to the wrong format it also was outdated.